.vscode/settings.json

```diff
@@ -10,32 +10,11 @@
   "eslint.format.enable": true,
```

Comment:
this settings is not in its right place (should be in its section below)

Reply:
will be doing a quick follow-up for any other revisions;
many other things often get stale here&hellip;